modules/apps/accessibility/accessibility-test/src/testFunctional/tests/AccessibilityMenu.testcase

```diff
@@ -61,25 +63,59 @@ definition {
 	}
 
 	@description = "LPS-178192. Verifies the modified accessibility configuration state is persisted in database for signed in user."
-	@ignore = "Test Stub"
 	@priority = 5
 	test ChangedConfigurationSettingByAuthenticatedUserWillBeSavedInDatabase {
 		property portal.acceptance = "true";
 
-		//task ("Given When Then") {
+		task ("Given created new user A") {
+			JSONUser.addUser(
+				userEmailAddress = "userea@liferay.com",
+				userFirstName = "userfnA",
+				userLastName = "userlnA",
+				userScreenName = "usersnA");
+		}
 
-		// // Given created new user A
-		// // When sign in as user A
-		// // And toggle show underline to OFF state)
-		// // And sign out
-		// // And guest user sets shows underline configuration to OFF
-		// // And guest user sets shows underline configuration to ON
-		// // And when sign in as user A
-		// // When open Accessibility Menu
-		// // Then show underline is set to OFF
+		task ("When sign in as user A") {
+			User.logoutAndLoginPG(
+				userLoginEmailAddress = "userea@liferay.com",
+				userLoginFullName = "userfnA userlnA");
+		}
 
-		//}
+		task ("And toggle an accessibility configuration (set underline to ON state)") {
+			AccessibilityMenu.openAccessibilityMenu();
+
+			Check.checkNotVisible(
+				key_toggleSwitchLabel = "Underlined Links",
+				locator1 = "ToggleSwitch#ANY_TOGGLE_SWITCH");
+		}
+
+		task ("And sign out") {
+			SignOut.signOut();
+		}
+
+		task ("And guest user sets shows Expanded Text configuration to ON") {
+			AccessibilityMenu.openAccessibilityMenu();
 
+			Uncheck.uncheckNotVisible(
+				key_toggleSwitchLabel = "Expanded Text",
+				locator1 = "ToggleSwitch#ANY_TOGGLE_SWITCH");
+		}
```

Comment:
Did you mean `underlined links` instead of `expanded text`?

Reply:
Ok, I corrected it.